examples/main/layer-controls.js

```diff
@@ -38,8 +38,8 @@ export default class LayerControls extends PureComponent {
 
   _renderSlider(settingName, value) {
     let max = 1;
-    if (/radius|width|height|pixel|size/i.test(settingName) &&
-      !(/^((?!scale).)*$/).test(settingName)) {
+    if (/radius|width|height|pixel|size|miter/i.test(settingName) &&
```

Comment:
This is getting a little hard on the eyes. Feels like we'll need a propTypes concept soon to avoid endlessely adding to the "guess list" here.